sympy/physics/control/tests/test_lti.py

```diff
@@ -1679,3 +1679,28 @@ def test_StateSpace_functions():
     assert ss3.input_matrix == Matrix([[0, 0], [1, 0], [0, 1], [0, 0]])
     assert ss3.output_matrix == Matrix([[0, 1, 0, 0], [0, 0, 1, 0]])
     assert ss3.feedforward_matrix == Matrix([[0, 0], [0, 1]])
+
+
+def test_Feedback_as_TransferFunction(): #Solves https://github.com/sympy/sympy/issues/26161
+    fd1 = Feedback(TransferFunction(s + 1, 1 , s), TransferFunction(s + 2, 1, s))
+    tf1 = TransferFunction(2 * (s+1), s**2 + 2*s + 1, s)
+    tf2 = TransferFunction(1, s + 1, s)
+    # Type Checking
+    assert isinstance(fd1, TransferFunction)
+    assert isinstance(fd1, Feedback)
+    assert not isinstance(tf1, Feedback)
+    # Operation checking
+    num, den = fd1.num, fd1.den
+    assert num == TransferFunction(s + 1, 1, s)
```

Comment:
Should we not actually return the numerator, returning a `TransferFunction` here will be useless as what the user wants with `.num` or `.den` is- 

```
>>> fd1 = Feedback(TransferFunction(s + 1, 1 , s), TransferFunction(s + 2, 1, s))
>>> fd1
Feedback(TransferFunction(s + 1, 1, s), TransferFunction(s + 2, 1, s), -1)
>>> fd1 = fd1.doit()
>>> fd1
TransferFunction(s + 1, (s + 1)*(s + 2) + 1, s)
>>> fd1.num
s + 1
>>> fd1.den
(s + 1)*(s + 2) + 1
```

Reply:
I think we can merge, once this is addressed.

Reply:
Yes, sir, I've included it in tests because in the original issue message here https://github.com/sympy/sympy/issues/26161#issue-2114036253
 It says that we can also call the .num and. den in the Feedback.

Reply:
> I'd also expect things like Feedback.num and Feedback.den to work. So can Feedback be a subclass of a TransferFunction

Reply:
Yes, I agree. The numerator should be the `symbolic expression` (numerator of the equivalent transfer function) instead of the `TransferFunction` is what I meant.

Reply:
I would like to have this behavior.
```
assert num == s + 1
assert den == (s + 1)*(s + 2) + 1
```

Reply:
I've added two properties to `Feedback` num and den.
I think, from this, it will follow this behaviour.